services/app-web/src/contexts/AuthContext.tsx

```diff
@@ -97,7 +97,7 @@ function AuthProvider({
             )
             modalCountdownTimers.current = []
         }
-    }, [sessionIsExpiring]) // full dep array causes a loop, because we're resetting the dep in the useEffect
+    }, [sessionIsExpiring, logoutCountdownDuration]) // full dep array causes a loop, because we're resetting the dep in the useEffect
```

Comment:
ooo just saw this comment seems like I need to lint ignore this one - will go back and do this